client/src/main/java/io/split/client/HttpSegmentChangeFetcher.java

```diff
@@ -69,42 +66,27 @@ public SegmentChange fetch(String segmentName, long since, FetchOptions options)
             }
 
             URI uri = uriBuilder.build();
-            HttpGet request = new HttpGet(uri);
 
-            if(options.cacheControlHeadersEnabled()) {
-                request.setHeader(CACHE_CONTROL_HEADER_NAME, CACHE_CONTROL_HEADER_VALUE);
-            }
-
-            response = _client.execute(request);
-
-            int statusCode = response.getCode();
-
-            if (_log.isDebugEnabled()) {
-                _log.debug(String.format("[%s] %s. Status code: %s", request.getMethod(), uri.toURL(), statusCode));
```

Comment:
For me, we don't should remove this log. You can leave this log on the debug level to know what happens when we fetch segments.

Reply:
The log is in the SplitHttpClientImpl class